[OSDOCS#17012]: CQA bare metal ipi workflow 2 (#117420)
openshift/openshift-docs: modules/ipi-install-setting-proxy-settings-within-install-config.adoc

[id='ipi-install-setting-proxy-settings-within-install-config_{context}']
= Setting proxy settings

[role="_abstract"]

> To deploy an {product-title} cluster while using a proxy, make the following changes to the `install-config.yaml` file.

Reviewer: Suggested edit to avoid self referential language:

```suggestion
You can deploy an {product-title} cluster while using a proxy by editing the `install-config.yaml` file.
```
